src/docs/compact.dox

```diff
@@ -2,11 +2,49 @@
 
 The WT_SESSION::compact method can be used to compact data sources.
 
-The data source does not have to be quiescent, compaction may be
-performed on an active data source.
+The data source does not have to be quiescent, compaction may be performed on an active data source.
 
-Because checkpoints named by the application are not discarded until
-explicitly removed or replaced, they may prevent WT_SESSION::compact
-from accomplishing anything.
+<b>Note:</b> Because checkpoints named by the application are not discarded until explicitly removed or replaced, they may prevent WT_SESSION::compact from accomplishing anything.
+
+@section foreground Foreground compaction
+
+Foreground compaction is executed when a specific table is given to the WT_SESSION::compact method.
+The call returns once compaction has been executed on the table.
```

Comment:
`table` to `specified uri`?

Reply:
Used URI in 47ae7c97c2ebca42ee19132458818f0ca1ade75f